src/Turn.js

```diff
@@ -1,12 +1,12 @@
-import Turn from './Turn.js';
+import Wheel from './Wheel.js';
 
```

Comment:
`import data from './data/sample-data.js';`

I think we might need this line in Turn to get the data similarly to Round.